README.md

```diff
@@ -1,69 +1,73 @@
+This is first project of The Veritable Computation Initiative. Our goal is to improve the tools developers use to write
+software. We're planning release a site with more information about Veritable soon.
+
+Passerine is currently being developed by [Isaac Clayton](https://github.com/slightknack), a high-school student with
+too much free time on his hands. A few people have offered feedback and suggestions from time to time. Huge thanks to
 [Raul](https://github.com/spaceface777),
 [Hamza](https://github.com/hhhapz),
 [Rishi](htps://github.com/rishiosaur),
 [Lúcás](https://github.com/cronokirby),
 [Anton](https://github.com/jesyspa/),
 [Yasser](https://github.com/realnegate),
-[Shaw](https://github.com/shawsumma)†,
+[Shaw](https://github.com/shawsumma)<sup>†</sup>,
 [Plecra](https://github.com/plecra),
 [IFcoltransG](https://github.com/IFcoltransG),
 [Jack](https://github.com/nivpgir),
-[Keith](https://github.com/Kethku)‡,
-Xal,
-and others!
+[Keith](https://github.com/Kethku)<sup>‡</sup>, Xal, and others!
 
-> † Shaw is writing an [alternative implementation of Passerine](https://github.com/ShawSumma/purr/tree/main/ext/passerine), and it's *super* fast. It's part of a wider effort of his to develop [an efficient language-agnostic VM](https://github.com/ShawSumma/purr).
-
-> ‡ Keith is currently [sponsoring](https://www.patreon.com/slightknack) the development of Passerine — I'm deeply grateful for his support!
+> <sup>†</sup>Shaw is writing an [alternative implementation of Passerine](https://github.com/ShawSumma/purr/tree/main/ext/passerine), and it's _super_ fast. It's part of a wider effort of his to develop [an efficient language-agnostic VM](https://github.com/ShawSumma/purr).<br/><sup>‡</sup> Keith is currently [sponsoring](https://www.patreon.com/slightknack) the development of Passerine — I'm deeply grateful for his support!
 
 ## An Overview
-Where this overview gets really exciting is when we dive into [macros](#macros). If you're here to give Passerine a try, [skip to Installation](#installation).
 
-> **⚠️ Note that Passerine is a *work in progress*: features mentioned in this overview may not be implemented yet.**
+Where this overview gets really exciting is when we dive into [macros](#macros). If you're here to give Passerine a
+try, [skip to Installation](#installation).
+
+> **:warning:Note that Passerine is a _work in progress_: features mentioned in this overview may not be implemented yet.**
 
 ### Syntax
-The goal of Passerine's syntax is to make all expressions as *concise* as possible while still conserving the 'feel' of *different types* of expressions.
+
+The goal of Passerine's syntax is to make all expressions as _concise_ as possible while still conserving the 'feel'
+of _different types_ of expressions.
```

Comment:
Two quick things, then I'll merge this PR:

1. I think your formatter added 80-column line breaks to the markdown file. Please remove them
2. The italics `*hello*` are still being changed to `_hello_`.